webhooks.py

```diff
@@ -16,8 +16,8 @@
 # under the License.
 
 import logging
-from sys import stderr, hexversion
-logging.basicConfig(stream=stderr)
+from sys import stdout, hexversion
+logging.basicConfig(stream=stdout, level=logging.INFO)
 
```

Comment:
I was trying to achieve some request logging on the screen to see what actually is comming, with stderr this was not possible from what I remember (with gunicorn)